### DISCUSSION TITLE
Question content encoded for... safety

### DISCUSSION THREAD

**Discussion by qwerfd2-wustl** [Lost-MSth/Arcaea-server#46] [Q&A]
QXJlIHRoZXJlIHdheXMgdG8gb2J0YWluIG9mZmljaWFsIHdvcmxkIG1hcHMgYmVzaWRlcyBmcm9tIG1lbW9yeSBkdW1waW5nIG9yIHJlY3JlYXRpbmcgdGhlIGZpbGUgYWNjb3JkaW5nIHRvIHRoZSBhcHAncyBiZWhhdmlvcj8=

**Comment by MBRjun** [comment]
安装包里可能会有？）

**Reply by qwerfd2-wustl** [reply]
Unfortunately no, these are from server requests (/years/19/world/map).
The thing is... a few of the maps provided are pretty accurate, so...

**Reply by qwerfd2-wustl** [reply]
Base64 helps when companies have automated filters for specific keywords (I believe GitHub have those as well). It's not intended as encryption.

**Comment by Lost-MSth** [comment]
VGhlIG9ubHkgd2F5IGlzIGNhdGNoaW5nIHRoZSBwYWNrYWdlcyBmcm9tIHRoZSBvZmZpY2lhbCBzZXJ2ZXIuIER1bXBpbmcgdGhlIG1lbW9yeSBvZiBtYXAgZGF0YSBpcyBqdXN0IG9uZSBtZXRob2Qgb2YgdGhpcyB3YXkuIFlvdSBjYW4gYWxzbyBkdW1wIHRoZSBzc2wgY2VydGlmaWNhdGUgZnJvbSB0aGUgY2xpZW50IGFuZCB1c2UgdGhlIGNlcnRpZmljYXRlIHRvIGNhdGNoIHRoZSBwYWNrYWdlcy4gV2VsbCwgbXkgc3VnZ2VzdGlvbiBpcyBjcmVhdGluZyBkaWZmZXJlbnQgbWFwcyBieSB5b3Vyc2VsZi4=

**Reply by MBRjun** [reply]
怎么现在都流行base64语了（（（（（（（（（（（（（（（（（（（

**Reply by Lost-MSth** [reply]
> 怎么现在都流行base64语了（（（（（（（（（（（（（（（（（（（

用什么问的就用什么回答(不过别是自编加密算法，套个五六层什么的

**Reply by qwerfd2-wustl** [reply]
Thank you, that made perfect sense（
Oh, and

**Reply by qwerfd2-wustl** [reply]
You mentioned before (in another issue) that... doing this... is easier on iOS than Android. Is that still true in the era of SSL verification?

**Reply by Lost-MSth** [reply]
> You mentioned before (in another issue) that... doing this... is easier on iOS than Android. Is that still true in the era of SSL verification?

I am not sure about this. I think remote mapping has equal difficulty in iOS and Android now.  However if you want to link to the official server via catching packages software, client ssl certificate is necessary, because it will be verified by the official server.

**Reply by qwerfd2-wustl** [reply]
I thought so. It shouldn't make a difference now as the latest systems across the board distrust user-added certificates anyways.
Oh, one more thing: WWVzdGVyZGF5LCBJIG1hbnVhbGx5IGNoYW5nZWQgdGhlIFVEUCBzZXJ2ZXIgaG9zdCBmb3IgbGlua2VkIHBsYXkgdG8gdGhlIG1haW4gc2VydmVyIGhvc3QuIEpvaW5pbmcgd29ya2VkLCBidXQgY3JlYXRpbmcgcm9vbSBzdGlsbCBoYXZlIHRoZSBlcnJvci4gSSdtIGEgbGl0dGxlIGNvbmZ1c2VkIG9uIHdoYXQgdGhpcyAibGluayBwbGF5IGhvc3QiIHNob3VsZCBiZS4=

**Reply by Lost-MSth** [reply]
QXJlIHRoZXJlIGFueSBlcnJvciBtZXNzYWdlcyBvbiB0aGUgc2VydmVyPyBUaGUgdW5rbm93biBlcnJvciBieSBjcmVhdGluZyB0aGUgcm9vbSBpcyByYXBpZCBvciBsYXp5IGVycm9yPwpUaGUgbGluayBwbGF5IGhvc3QgaXMganVzdCBmb3Igc29tZSBzZXJ2ZXJzIHdoaWNoIHVzZSBuZ2lueCwgQ2hhcmxlcywgb3IgYW55IG1pZGRsZSBzZXJ2ZXIsIHdoaWNoIGNhbiBub3QgZ2V0IHRoZSByaWdodCBob3N0LiBZb3UgY2FuIGNoZWNrIHRoaXMgYnkgcHJpbnQgdGhlIGhvc3QodmFsdWVbJ2VuZHBvaW50J10gSSByZW1lbWJlcikgaW4gYHVkcF9tYWluLnB5YC4=

**Reply by qwerfd2-wustl** [reply]
There are no error message from the server. The "unknown error" is from the client, about 5 seconds after the server sends back the create success message. Could it be that my wifi (which is not under my control) blocked certain traffic?

**Reply by Lost-MSth** [reply]
Okay I know. It is a lazy error so I guess the client can not send a UDP package to the server. You mentioned 'wifi'. Emmm...I want to know which way you used to connect to the local server.
SWYgeW91IHVzZSBDaGFybGVzLCBhbiBhcHAgd2hpY2ggY2FuIGJ1aWxkIFVEUCBzb2NrcyBjb25uZWN0aW9ucyBpcyBuZWVkZWQu

**Reply by qwerfd2-wustl** [reply]
SSdtIHVzaW5nIHZwcm94aWQoc29ja3M1LCBhbmRyb2lkKSArIGNoYXJsZXMod2luZG93cykuIFNvIEkgc2hvdWxkIGZpbmQgYW4gYXBwIG9uIFdpbmRvd3MgdGhhdCBjcmVhdGUgYSBzb2NrcyBjb25uZWN0aW9uIGJhY2sgdG8gQW5kcm9pZD8=

**Reply by Lost-MSth** [reply]
VnByb3hpZD8gT2ggaXQgZG9lc24ndCBzdXBwb3J0IFVEUCBzb2Nrcy4gSSB3YXMgdHJvdWJsZWQgYnkgdGhpcyBiZWZvcmUgdG9vLiBTbyBJIHJlY29tbWVuZCAnUG9zdGVybicgbm93LiBPZiBjb3Vyc2UgeW91IGNhbiB1c2UgdjJyYXkgb3IgYW55dGhpbmcgZWxzZS4=

**Reply by qwerfd2-wustl** [reply]
T2gsIHJpZ2h0LiBJIHRyaWVkIFJQcm94aWQgb24gbXkgcm9vdGVkIHBob25lIGFuZCBldmVyeXRoaW5nIHdvcmtlZC4gVG9vIGJhZCB0aGF0IG15IHRhYmxldCBpcyBub3Qgcm9vdGVkIC0gSSB0cmllZCBwb3N0ZXJuIGFuZCB2MnJheSwgYnV0IGEgbGFjayBvZiBhcHAgd2hpdGVsaXN0IHJlbmRlcnMgdGhlIGRldmljZSBvZmZsaW5lIChmb3IgYXBwcyBvdGhlciB0aGFuIGFyY2FlYSkgd2hlbiBjb25uZWN0ZWQuIEl0IHNlZW1zIHRvbyBiaWcgb2YgYSBzYWNyaWZpY2UgYXMgbm8gb25lIHdpbGwgZXZlciBqb2luIGxpbmsgcGxheSBhbnl3YXlzIChpdCdzIGp1c3QgbXlzZWxmIFF3USkuIFBlcnNvbmFsbHkgaXQncyBub3Qgd29ydGggaXQgKQ==

**Reply by Lost-MSth** [reply]
SXQgc2VlbXMgcGVyZmVjdCBhbmQgaXQgaXMgYWxzbyBwaXR5IGZvciBubyBvbmUgdG8gcGxheSB0b2dldGhlci4KV2VsbCwgSSBhbSBjb25mdXNlZCB0aGF0IHBvc3Rlcm4gZG9lc24ndCBuZWVkIFJPT1QgYXQgYWxsLiBTZXQgdGhlIHJ1bGUgdGhhdCBvbmx5IGhvc3QgJ2FyY2FwaV92Mi5sb3dpcm8uY29tJyBjYW4gY29ubmVjdCB2aWEgc29ja3MgcHJveHkgYW5kIHRoZW4gdGhlIG90aGVyIGFwcHMgd2lsbCBub3QgYmUgYWZmZWN0ZWQu

**Reply by qwerfd2-wustl** [reply]
T2ggdGhhdCdzIHJlYWxseSBjb29sISBBZGRlZCB0aGUgcnVsZSBhbmQgcG9zdGVybiBpcyB3b3JraW5nIG5vdy4gVG9vIGJhZCB0aGF0IGFuZHJvaWQgd2lsbCBjbG9zZSB0aGUgYXBwIGFmdGVyIGEgd2hpbGUgKG15IE9TIGhhdmUgcmVhbGx5IHN0cmljdCBwb3dlciBtZ210KS4gdGJoIEkgd2lsbCBzdGljayB3aXRoIHZwcm94aWQgZm9yIG5vdyBhcyBJJ20gYWxvbmUgb3V0IGhlcmUgKEFyY2FlYSBpcyB2ZXJ5IG5pY2hlIHdoZXJlIEkgbGl2ZSkuIFRoYW5rIHlvdSBzbyBtdWNoIQ==
